### ISSUE TITLE
about MarginMSELoss

### ISSUE THREAD

**Issue by ScottishFold007** [UKPLab/sentence-transformers#1111]
Hello! I saw the other day that you added a new loss function - MarginMSELoss, I am interested in it, in what scenario should I use it? Thanks~

**Comment by nreimers** [comment]
It implements the loss here:
https://arxiv.org/abs/2010.02666

It is quite useful when you want to distill knowledge from a CrossEncoder to a BiEncoder.

Currently we run several experiments with it and get quite strong results for semantic search models: MSMARCO model got a 37 MRR@10 compared to 34 MRR@10 for the TAS-B model (https://www.sbert.net/docs/pretrained-models/msmarco-v3.html)

More details will be published in a paper in September.
